micrometer-shim/src/main/java/io/opentelemetry/example/micrometer/Controller.java

```diff
@@ -0,0 +1,13 @@
+package io.opentelemetry.example.micrometer;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class Controller {
+
+  @GetMapping("/ping")
```

Comment:
Nit: how about adding
```suggestion
  @GetMapping("/ping")
  @Timed("ping.time")
```